NSS-Day-Cohort-53/tabloidcli-bubbling-bullfrogs
# Delete tags

#### Changes Made
1. Modified file `PostDetailManager.cs` to include `RemoveTag()` method which presents a list of tags to delete and uses `DeleteTag()` method from `PostRepsoitory.cs` to delete selected tag.
2. Modified `PostRepositiory.cs` to include `DeleteTag()` that deleted selected Post Tag from PostTag table
3. Modified  the `Delete()` method in `PostRepository.cs` to cascade delete PostTags when Posts are deleted
4. Modified the `Delete()` method in `BlogRepository.cs` to cascade delete BlogTags, and Posts when Blogs are deleted.
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout Sh-DeleteTags
    ```
5. Open a new Terminal tab (⌘T) and navigate to the server directory.
6. Test app functionality.
    > Test removing tags and blogs in Blog Manager and Post Details.
7. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
